code/game/smoothwall.dm

```diff
@@ -1,112 +1,87 @@
-		for(var/obj/machinery/door/W in orange(src,1)) //doors should not result in diagonal walls, it just looks ugly. checking if area is shuttle so it won't merge with the station
-			if((abs(src.x-W.x)-abs(src.y-W.y)) && (istype(W.loc.loc,/area/shuttle) || istype(W.loc.loc,/area/supply)))
-				junction |= get_dir(src,W)
-		for(var/obj/structure/grille/W in orange(src,1)) //same for grilles. checking if area is shuttle so it won't merge with the station
-			if((abs(src.x-W.x)-abs(src.y-W.y)) && (istype(W.loc.loc,/area/shuttle) || istype(W.loc.loc,/area/supply)))
-				junction |= get_dir(src,W)*/
-
-	if(istype(src,/turf/simulated/wall))
-		var/turf/simulated/wall/wall = src
-		wall.icon_state = "[wall.walltype][junction]"
-	else if (istype(src,/obj/structure/falsewall))
-		if(istype(src,/obj/structure/falsewall/reinforced))
-			icon_state = "rwall[junction]"
-		else
-			var/obj/structure/falsewall/fwall = src
-			fwall.icon_state = "[fwall.mineral][junction]"
-/*	else if(istype(src,/turf/simulated/shuttle/wall))
-		var/newicon = icon;
-		var/newiconstate = icon_state;
-		if(junction!=5 && junction!=6 && junction!=9 && junction!=10) //if it's not diagonal, all is well, no additional calculations needed
-			src.icon_state = "swall[junction]"
-		else //if it's diagonal, we need to figure out if we're using the floor diagonal or the space diagonal sprite
-			var/is_floor = 0
-			for(var/turf/unsimulated/floor/F in orange(src,1))
-				if(abs(src.x-F.x)-abs(src.y-F.y))
-					if((15-junction) & get_dir(src,F)) //if there's a floor in at least one of the empty space directions, return 1
-						is_floor = 1
-						newicon = F.icon
-						newiconstate = F.icon_state //we'll save these for later
-			for(var/turf/simulated/floor/F in orange(src,1))
-				if(abs(src.x-F.x)-abs(src.y-F.y))
-					if((15-junction) & get_dir(src,F)) //if there's a floor in at least one of the empty space directions, return 1
-						is_floor = 1
-						newicon = F.icon
-						newiconstate = F.icon_state //we'll save these for later
-			for(var/turf/simulated/shuttle/floor/F in orange(src,1))
-				if(abs(src.x-F.x)-abs(src.y-F.y))
-					if((15-junction) & get_dir(src,F)) //if there's a floor in at least one of the empty space directions, return 1
-						is_floor = 1
-						newicon = F.icon
-						newiconstate = F.icon_state //we'll save these for later
-			if(is_floor) //if is_floor = 1, we use the floor diagonal sprite
-				src.icon = newicon; //we'll set the floor's icon to the floor next to it and overlay the wall segment. shuttle floor sprites have priority
-				src.icon_state = newiconstate; //
-				src.overlays += icon('icons/turf/shuttle.dmi',"swall_f[junction]")
-			else //otherwise, the space one
-				src.icon_state = "swall_s[junction]"*/
-
-	return
+// OKAY I DON'T KNOW WHO THE FUCK ORIGINALLY CODED THIS BUT THEY ARE OFFICIALLY FIRED FOR BEING DRUNK AND STUPID
+// FUCK YOU MYSTERY CODERS
+// FOR THIS SHIT I'M GOING TO MAKE ALL MY COMMENTS IN CAPS

```

Comment:
well I did in the PR~ Haha
It's kinda nice for people who don't know the story to be like "Wow, this guy's angry!"